salt/metalk8s/addons/monitoring/kube-state-metrics/deployed.sls

```diff
@@ -219,11 +219,11 @@ spec:
         name: kube-state-metrics
```

Comment:
Add information in the commit message (you can copy your PR description if you want, in fact when you have a single commit, its message is used as PR description ;))